Scripts/.flake8

```diff
@@ -19,10 +19,16 @@ ignore =
     # D105	Missing docstring in magic method
     # D107	Missing docstring in __init__
     # TODO: Fix some of these?
-    D101,D102,D103,D105,D107
+    D101,D102,D103,D105,D107,
     # ====================================
     # pydocstyle: D### - Whitespace Issues
     # ====================================
     # D412	No blank lines allowed between a section header and its content
     # We ignore D412 deliberately:
-    D412
+    D412,
+    # ================================================
+    # flake8 whitespace: W50#
+    # ================================================
+    # W503	Line break before binary operator
+    # PEP8 is now in favour of line-breaks before binary operators
+    W503
```

Comment:
Can you put this at line 10 of the file with the other flake8 built in codes please?